### PR TITLE
Deploy scripts

### PR DESCRIPTION
**Pull request by mariogutval**
Add deploy script and .env.default files.

TODO:

- [ ] Deployment tests
- [x] Deploy on Goerli

### CODE REVIEW

File: migrations/1_deploy_contracts.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const { deployProxy } = require("@openzeppelin/truffle-upgrades")
+const { Address } = require("ethereumjs-util")
+
+module.exports = async function (deployer) {
+  const flag = 0
+	const ZERO_ADDRESS = Address.zero().toString()
+  const proxyContract = await deployProxy(
+    CMTAT_PROXY,
+		[
+			"0x73E3552Cdbe9F4F38A1dDf1262b87Aa208d2225C",
```
Comment (diego-G) [inline migrations/1_deploy_contracts.js]: Better to write this as a parameter.

File: truffle-config.js [hunk 2]
```diff
@@ -34,6 +41,51 @@ module.exports = {
...
-    'truffle-contract-size'
-  ]
+    'truffle-contract-size',
+    'truffle-plugin-verify'
+  ],
+  api_keys: {
+    etherscan: process.env.ETHERSCAN_API_KEY,
+  },
 }
+// export const networks = {
```
Comment (diego-G) [inline truffle-config.js]: Shall we remove all this?

File: truffle-config.js [hunk 3]
```diff
@@ -34,6 +41,51 @@ module.exports = {
     }
   },
   plugins: [
-    'truffle-contract-size'
-  ]
+    'truffle-contract-size',
+    'truffle-plugin-verify'
```
Comment (diego-G) [inline truffle-config.js:45]: Is it finally working?

**Review by diego-G — CHANGES_REQUESTED** [review]

File: truffle-config.js (hunk 3 above)
Comment (mariogutval) [reply truffle-config.js:45]: Yeah, it works "manually"

**Review by diego-G — APPROVED** [review]

**Pull request merged**